packages/admin-ui/src/table/resolvers/image.tsx

```diff
@@ -18,6 +20,12 @@ const defaultPreview: ImagePreview = {
 }
 
 function getImageVariant(density: TableDensity): StyleObject {
+  const outlineStyle = style({
+    outlineColor: (theme) => get(theme, 'bg.primary', 'bg.primary'),
+    outlineWidth: '0.125rem',
+    outlineStyle: 'solid',
+  })
+
   return {
     regular: {
```

Comment:
Similar to this [case](https://github.com/vtex/admin-ui/pull/265/files#r906314635). I will set the variable style as the base style of the image.